apps/artsy_primer/routes.js

```diff
@@ -0,0 +1,11 @@
+import Article from '../../models/article'
+import embed from 'particle'
+import { crop } from '../../components/resizer'
+
+export const article = async (req, res, next) => {
+  const article = new Article({id: req.params.id})
+  await article.fetchWithRelated({
```

Comment:
Using ES6 here which gives us [async/await](https://www.twilio.com/blog/2015/10/asyncawait-the-hero-javascript-deserved.html) ❤️